packages/dev/docs/pages/react-spectrum/index.mdx

```diff
@@ -16,6 +16,10 @@ import heroNarrow2x from 'url:../assets/hero-narrow@2x.png';
 import heroWide from 'url:../assets/hero-wide.png';
 import heroWide2x from 'url:../assets/hero-wide@2x.png';
 
+---
+description: A React implementation of Spectrum, Adobe’s design system. Spectrum provides adaptive, accessible, and cohesive experiences for all Adobe applications.
```

Comment:
some of these are copies of the first line, some are that + more, should the more ones be added to the visible text?

Reply:
i think it should be fine...since the second sentence is more a summary of the page, which is already visible (and has more detail). this is more for robots to index things + og data when sharing links.